Library/.rubocop.yml

```diff
@@ -28,7 +28,7 @@ FormulaAuditStrict/Options:
   Enabled: true
 
 NewFormulaAudit/Options:
-  Enabled: false
+  Enabled: true 
```

Comment:
@GauthamGoli I think we need to have something that involves zero configuration. Is there any way to have a cop that's only run when specified on the command-line? There may be some hack using a global `ARGV` check or similar in the cop itself.